core/router.js

```diff
@@ -17,7 +17,7 @@
 var bloomrun = require('bloomrun')
 var assert = require('assert')
 var stringify = require('fast-safe-stringify')
-var errors = require('./err')
+var mue = require('mu-error')({dev: process.NODE_ENV !== 'production'})
```

Comment:
hah - yes agreed - I just did that before I looked at this. 

That allows for `mu.error` to be used by users of mu